wordpress-mobile/WordPress-iOS
# Update GridIcons to v 0.20-beta.1

Ref https://github.com/Automattic/Gridicons-iOS/pull/45

Updates GridIcons to v 0.20-beta.1 (supports Swift 5)
#### Notes
- will be merged once the new pod version is ready


To test:

- build/run
- check that icons are displayed correctly in any part of the app (e.g., reader, editors, etc)

PR submission checklist:
